twitter/samples/twitter/twitter_test.bal

```diff
@@ -0,0 +1,89 @@
+import ballerina.lang.system;
+import ballerina.lang.messages;
+import ballerina.net.http;
+import ballerina.lang.jsons;
+import org.wso2.ballerina.connectors.twitter;
+import ballerina.test;
+
+string tweetId;
+
+function init()(twitter:ClientConnector) {
+        string consumerKey = system:getEnv("CONSUMER_KEY");
+        string consumerSecret = system:getEnv("CONSUMER_SECRET");
+        string accessToken = system:getEnv("ACCESS_TOKEN");
+	string accessTokenSecret = system:getEnv("ACCESS_TOKEN_SECRET");
+	twitter:ClientConnector twitterConnector = create twitter:ClientConnector(consumerKey, consumerSecret, accessToken, accessTokenSecret);
+        return twitterConnector;
+
+}
+
+
+function testTweet() {
+        twitter:ClientConnector twitterConnector = init();
+        message response = twitter:ClientConnector.tweet(twitterConnector, "#ballerina");
+        json jsonResponse = messages:getJsonPayload(response);  
+        int status = http:getStatusCode(response);
+	string value = jsons:getString(jsonResponse,"text");
```

Comment:
shall we remove jsons functions from here? see the below example
json tmp = {"sample":{"saman":"name"}};
string val = tmp.sample.saman;